<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_NewProject</class>
 <widget class="QMainWindow" name="Dialog_NewProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Project</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="label_NewProject_ProjectDescription">
        <property name="text">
         <string>Project Description</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDialogButtonBox" name="buttonBox__NewProject">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_NewProject_ProjectAuthor">
        <property name="text">
         <string>Author</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_NewProject_LoadCSV">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_NewProject_Project1gVocabName">
        <property name="text">
         <string>Vocab 1gram name</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>* needed value</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="pushButton_NewProject_LoadCSV">
        <property name="text">
         <string>Load CSV*</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_NewProject_ProjectName">
        <property name="text">
         <string>Projet Name*</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_NewProject_ProjectName"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_NewProject_ProjectAuthor"/>
      </item>
      <item row="2" column="1">
       <widget class="QTextBrowser" name="textBrowser_NewProject_ProjectDescription"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_NewProject_ProjectNgVocabName">
        <property name="text">
         <string>Vocab Ngram name</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_NewProject_Project1gVocabName">
        <property name="text">
         <string>vocab1g</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_NewProject_ProjectNgVocabName">
        <property name="text">
         <string>vocabNg</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>489</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
